CHANGELOG.md

```diff
@@ -122,7 +147,7 @@
 - Replaced the deprecated `dbt.current_timestamp_backcompat()` function with `dbt.current_timestamp()` to ensure all timestamps are captured in UTC.
 - Updated `fivetran_platform__connector_daily_events` to support running `dbt compile` prior to the initial `dbt run` on a new schema.
 
-# dbt_fivetran_log v1.9.0
+# dbt_fivetran_log v1.9.5
```

Comment:
Was this change intentional? Shouldn't it still be v1.9.0 since it's the release of the package and not the dbt-core ref?